src/popup/router/components/Modal.vue

```diff
@@ -179,6 +179,10 @@ $modal-bg-color: variables.$color-bg-4;
       padding-top: 0;
       color: variables.$color-light-grey;
       word-break: break-word;
+
+      &.no-padding {
+        padding-bottom: 0;
+      }
```

Comment:
Can be removed
```suggestion
```

Reply:
no becouse this is in the same level as .body